cachix/src/Cachix/Client/NetRc.hs

```diff
@@ -16,9 +16,9 @@ import           Cachix.Client.Config (Config, authToken)
 import           Cachix.Client.Exception (CachixException(NetRcParseError))
 
 
-{- Add a list of binary caches to netrc under `filename`.
-   Makes sure there are no duplicate entries (using domain as a key).
-   If file under filename doesn't exist it's created.
+{- | Add a list of binary caches to netrc under `filename`.
+   | Makes sure there are no duplicate entries (using domain as a key).
```

Comment:
Doesn't that introduce `|`s in the output?

I think we should standardize on
```
-- |
--
```

Similar problem here: https://hackage.haskell.org/package/cachix-0.1.1/docs/Cachix-Client-URI.html